app/src/main/res/layout/activity_auth_setup_biometrics.xml

```diff
@@ -63,11 +63,11 @@
                     android:layout_marginTop="20dp"
                     android:contentDescription="@string/auth_setup_biometrics_image"
                     android:src="@drawable/ic_fingerprint"
-                    android:tint="@color/theme_white"
                     app:layout_constraintBottom_toTopOf="@+id/button_layout"
                     app:layout_constraintLeft_toLeftOf="parent"
                     app:layout_constraintRight_toRightOf="parent"
-                    app:layout_constraintTop_toBottomOf="@+id/instruction_textview" />
+                    app:layout_constraintTop_toBottomOf="@+id/instruction_textview"
+                    app:tint="@color/theme_white" />
```

Comment:
Was this also an error (android:tint vs. app:tint) or is it just the lint that you are addressing?

Reply:
android:tint has some compatibility issues https://developer.android.com/reference/android/widget/ImageView#attr_android:tint

Reply:
Right, but was it necessary to change? Is it certain that nothing is affected by this?

Looking at documentation it looks like that should be the case, but I also saw some issues reported.

Reply:
The app will build with it, but it's reported as error in Android Studio when you open the preview and the recommended solution is to replace it with app:tint